src/base-element.js

```diff
@@ -503,8 +530,8 @@ export class BaseElement {
    * @param {!Element|!Array<!Element>} elements
    * @protected
    */
-  schedulePause(elements) {
-    this.resources_.schedulePause(this.element, elements);
+  scheduleUnload(elements) {
```

Comment:
Here's the specific scenario that @erwinmombay was supporting here: it's a carousel and one of the slides is the video and the user has swiped to the next slide. The immediate goal was to let the carousel to pause the video. Unlayout for this seems a bit extreme, no?